test/IonDecimal.ts

```diff
@@ -171,7 +175,7 @@ let decimalComparisonTests = [
 ];
 
 let decimalEqualsTests = [
-    {input1: '0', input2: '0', expected: true},
+    {input1: '0', input2: '0', expected: true, skip: false},
```

Comment:
Not sure why we have `skip: false` here? Could you add a comment.

Reply:
Typescript is able to infer a (pretty verbose) data type for `decimalEqualsTests` based on the fields/types found in the object literals in the array declaration.

Previously, some of the tests needed to be skipped, so they included a `skip: true` field. This PR obviated the need to skip them, but when I removed `skip: true`, I found that Typescript would complain about `skip: boolean` not being part of `decimalEqualsTests`'s data type when I used object destructuring:

```typescript
    describe('equals', () => {
        decimalEqualsTests.forEach(
            ({input1, input2, expected, skip = false}) => { // <--- Object destructuring
               // ...
                if(skip) {
                    // ...
                } else {
                    // ...
                }
            }
        );
    });
```

Adding `skip: false` to the first test as an example fixed the issue.

It's difficult to find documentation that describes this combination of behaviors that I could link to in a comment, and as you can see it takes a fair bit of text to explain the issue. 😛 I've opted to explicitly declare `decimalEqualsTests`' data type, allowing me to remove `skip: false`.

```typescript
let decimalEqualsTests: ({input1: string, input2: string, expected: boolean, skip?: boolean})[] = [
    // ...
];
```
Hopefully that's a bit more self-explanatory.

Reply:
This seems to be an issue with Typescripts inference engine. Seems like a bug to me if the inferred type relies on the first instance of a value and not the rest. I would expect a union type for the last field `skip`.

Explicitly pinning the type for the variable is the sane thing to do here. 
Thanks